site/journal/2020/Wrapping Up 2020 and Starting 2021.md

```diff
@@ -0,0 +1,185 @@
+
+If I can figure out how to set up a pull-up bar, I’ll add a pull-up count to this list as well! (It’s weirdly difficult to find a good spot in my house for a pull-up bar!) And of course I’ll be aiming to shed those 5 pounds if I can manage it, but even more urgently to do as I have the last few years and avoid *gaining* any weight. In all of these goals, my biggest concern is to be healthy—sustainably healthy, for the rest of my life.
+
+## Work
+
+There are two major things I have to cover in this section, and they’re *very* different from each other: LinkedIn (going great!) and <b><i>re</i>write</b> (…not so much).
+
+### LinkedIn
+
+I’m now wrapping up my second year at LinkedIn, and I’m delighted to report that it remains a *fantastic* fit for me. I’ve told Jaimie over and over again how happy it makes me to be here: a place where I feel like I can make real contributions that are well-matched to my abilities and interests, and where those contributions are recognized and valued. In every other job I’ve had, by the time I had hit the two-year mark, I was deeply frustrated and trying to figure out how long I could endure before bailing. With LinkedIn… not so much. I’m not even tired of my current *team* yet, and there are lots of opportunities for lateral motion internally when I *do* get there.
+
+Working at this scale—a multiple-orders-of-magnitude jump from my previous role—has had its challenges and frustrations, but in so many ways I feel like this is what I was made for, at least at this point in my career.
+
+At the end of 2019, I wrote:
+
+> If 2019 was the year of getting my feet under me at LinkedIn, I’m hoping that 2020 will be a year of really starting to make an impact.… I want to continue growing as a mentor to those more junior engineers. I want to expand my influence and leadership both within my own organization and in cross-cutting efforts. I want to deliver on a technical front the things LinkedIn needs to continue excelling — both by executing on initiatives other leaders have already identified as important, and by identifying and executing on initiatives I myself identify as important.
+
+Happily, I can say this year was a success to varying degrees on all those fronts.
+
+I have moved even more into a position of leadership within the front-end community at LinkedIn. In summer 2019, I identified that Ember Octane was *the* I needed to focus on critical technical investment for the LinkedIn.com app, and I have spent nearly all my time since then doing whatever was necessary to get us there—whether that meant spending 8 months working on upgrading the app through from Ember 3.4 to 3.8 to 3.12 to 3.16, writing documentation and mentoring developers on the new idioms, or coordinating the launch of a cross-cutting initiative in which every team on the app is committed to finishing the conversion over the next few quarters.
```

Comment:
@chriskrycho Ember octane was *the* what you needed to focus on?

I love reading your writing. Blessed new year to you and your family! 🎉

Reply:
Ahh! Thank you for the catch! I’ll have a fix up in five minutes or so. And grace and peace to you and yours!

Reply:
See a8463d9852564feddfa4f2402fd9c684d213b268!